core/testing/src/main/kotlin/com/google/samples/apps/nowinandroid/core/testing/repository/TestUserDataRepository.kt

```diff
@@ -112,9 +110,7 @@ class TestUserDataRepository : UserDataRepository {
         }
     }
 
-    /**
-     * A test-only API to allow setting of user data directly.
-     */
+    @TestOnly
```

Comment:
Testing module consist of bundle of test source.
I'm not sure ```@TestOnly``` needs in here.
Just my opinion.

Reply:
@Jaehwa-Noh Thanks for the feedback 😀

One could think that providing duplicate information might be unnecessary.
The change to `@TestOnly` was made to simplify and explicitly state what had previously been indicated through comments.

Reply:
`@VisibleForTesting` can be used

_Sent from my 2201117TG using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
My understanding of the differences between each annotation is as follows:

- `@VisibleForTesting` is used to explicitly state that the visibility of an element has been increased for testing purposes. In other words, while a lower visibility (e.g., private) would suffice for the actual app operation, the visibility has been temporarily raised to allow test code to access that element.
- `@TestOnly` indicates that specific code should be used in the test environment only. Thus, by using this annotation, it is explicitly stated that the code should not be used in actual production code.

@hoc081098  Am I misunderstanding anything? 🤔

Reply:
Can we refactor this so that the repository can be tested without using this API?

Reply:
@JoseAlcerreca Okay, I'll try 😀